catlin/pkg/validator/validate_test.go

```diff
@@ -35,6 +35,7 @@ metadata:
   annotations:
     tekton.dev/tags: a,b,c
     tekton.dev/pipelines.minVersion: "0.12"
+    tekton.dev/category: "Example"
```

Comment:
test is failing locally for this case. Can you please check
/hold

Reply:
Updated the change thanks!